929 log2

ChrisMcGowan opened the pull request:
## Changes proposed in this pull request:
- Add 9/29 change log

<!-- Replace "BRANCH_NAME" in the following URL before you submit this PR. -->
:sunglasses:[PREVIEW URL](https://cg-88d42ca6-59d7-47e0-9500-4dd9251360b9.app.cloud.gov/preview/cloud-gov/cg-site/929-log2)


## Security Considerations
n/a

Commit:
adding change log for 9292022

Commit:
Merge branch 'master' of github.com:cloud-gov/cg-site into 929-log
Adding in updated link for link checker

ChrisMcGowan committed:
Update 2022-09-29-release-notes.md

removed broken pivotal tracker links

dandersonsw commented on _posts/2022-09-29-release-notes.md:
```diff
@@ -0,0 +1,190 @@
+### Java Buildpack - v4.52 up from v4.50
+
+* Deprecation of Spring Cloud Connectors & Spring Auto Reconfiguration - this feature is now disabled by default
+* See release notes for latest versions of depenancies - https://github.com/cloudfoundry/java-buildpack/releases/tag/v4.52
```
```suggestion
* See release notes for latest versions of dependencies - https://github.com/cloudfoundry/java-buildpack/releases/tag/v4.52
```

ChrisMcGowan committed:
Update _posts/2022-09-29-release-notes.md

Co-authored-by: David Anderson <78759746+dandersonsw@users.noreply.github.com>

ChrisMcGowan committed:
Update .lycheeignore

Congress site now using Cloudflare and requires cookies and javascript so link checker fails
